pipes-and-filters/PipesAndFilters.Shared/ServiceBusPipeFilter.cs

```diff
@@ -98,19 +103,20 @@ public async Task Close(TimeSpan timespan)
             //  We simply stop any new processing, wait for existing thread to complete, then close the message pump and then return
             Thread.Sleep(timespan);
 
-            this.inQueue.Close();
+            await this.processor.CloseAsync();
 
             // Cleanup resources.
-            await ServiceBusUtilities.DeleteQueueIfExistsAsync(Settings.ServiceBusConnectionString, this.inQueue.Path);
+            await ServiceBusUtilities.DeleteQueueIfExistsAsync(Settings.ServiceBusConnectionString, this.inQueue.FullyQualifiedNamespace);
```

Comment:
I think this should be EntityPath rather than FullyQualifiedNamespace. Do we have a way to test this?